pkgs/data/fonts/vollkorn/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib, stdenv, fetchzip }:
+
+stdenv.mkDerivation rec {
```

Comment:
fonts usually only use fetchurl/fetchzip and postFetch.

Reply:
I took the `comic-neue` derivation as an inspiration.
Using the install phase to move files to their final dedicated places is more true to the semantics of that action IMHO, as that's the actual definition of install. But of course I can move that t postFetch if required.